src/__tests__/Error.test.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
```

Comment:
- [ ] Please fix this error as the previous reviewer requested: 
![image](https://github.com/mohamedgamaleldin999999/metrics-app/assets/91486791/67db4f8a-6b9e-479b-a619-d2f6e647e43d)